src/config.ts

```diff
@@ -1,4 +1,33 @@
-export const Config = {
+import { z } from "zod";
```

Comment:
TODO: split the types here into a separate file, and add comments explaining that config here is used only for Wrangler deployment.